routes/doctors.js

```diff
@@ -0,0 +1,44 @@
+var express = require("express");
+var router = express.Router();
+import DB from "../scripts/DB";
+
+/** GET All Doctors*/
+router.get("/", function (req, res) {
+  DB.retrieveAllDoctors(function(data) {
+    res.header("Content-Type",'application/json');
+    res.send(JSON.stringify(data, null, 4));
```

Comment:
You can use `res.json({})`. It will set the headers content type as application/json by default and you dont have to worry about stringifying json